api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -192,13 +192,16 @@ function undelegationReducer(undelegation, address, network) {
   }
 }
 
-function transactionsReducerV2(network, extrinsics, blockHeight, reducers) {
+async function transactionsReducerV2(network, extrinsics, blockHeight, db) {
```

Comment:
why remove reducers from here? I think it is good to have then here. It is not only `coinReducer`, there could be more we want to use. But ok, not so important

Reply:
In reducers you already have access to all reducer functions directly, don't need to carry.